liblouis/logging.c

```diff
@@ -94,8 +94,13 @@ static logLevels logLevel = LOU_LOG_INFO;
 static int logLevelSet = 0;	 // whether the log level has been set, through
 							 // lou_setLogLevel() or lou_setLogLevelFromEnvironment()
 
-void EXPORT_CALL
-lou_setLogLevelFromEnvironment() {
+/**
+ * Get the log level from the environment.
+ *
+ * Queries the environment (`LOUIS_LOGLEVEL`) once for the log level.
+ */
+void
+_lou_getLogLevelFromEnvironment() {
```

Comment:
This should be changed to a static function again. (I guess you can just cherry-pick https://github.com/liblouis/liblouis/pull/1816/commits/8c57ad3fbb5ba6f3345641fa4bd5de8a60cc58cd again.)